app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, filename_from, filename_to = command.split()
```

Comment:
```suggestion
    operation, filename_from, filename_to = command.split()
```
check if the operation is equal to "cp"